modules/apps/frontend-theme/frontend-theme-font-awesome-web/bnd.bnd

```diff
@@ -1,4 +1,4 @@
 Bundle-Name: Liferay Frontend Theme Font Awesome Web
 Bundle-SymbolicName: com.liferay.frontend.theme.font.awesome.web
-Bundle-Version: 3.0.9
+Bundle-Version: 3.0.10
```

Comment:
I don't think we need to update module bundle version, only `gradle baseline` does this.

See for example [the latest Clay update](https://github.com/liferay-frontend/liferay-portal/pull/1291/commits/b06017e8ec45be3fbd6d5aa12b5779ea55951fca), we are only updating dependency versions.

Reply:
So, is this different? https://github.com/liferay/liferay-portal/commit/3630cd580a7f8b411b8aa6708b824ff936fc8498

Reply:
It looks like Brian ran a script that upgraded micro versions of all modules. You can see thousands of these 'prep next' commits. I don't know why, maybe it's needed for GA release?

Reply:
Brian does that automatically before every GA release (in fact for every time we try to do a release) to publish a new version. The reason is because `frontend-theme-font-awesome` using the `liferay-font-awesome` 1.0.0 dependency is not the same as the one using the 1.0.1 version. 

If we change the version number in every dependency change we would bump the version number a lot without releasing anything. If we just do it before the release we consolidate changes and get less changes in version numbers. 

As a rule we don't have to update version numbers unless `gradle baseline` suggest otherwise.